lib/chrome.js

```diff
@@ -397,6 +398,7 @@ class ChromeTab {
     let isToGateway = gatewayUrl && request.url.indexOf(gatewayUrl) >= 0
     if (!this.manifest || !isToGateway) {
       return this.cdp.Fetch.continueRequest({ requestId })
+          .catch(err => { console.error('Non-fatal error from cdp.Fetch.continueRequest', err) })
```

Comment:
I was able to determine that this call specifically is related to https://github.com/superhuman/desktop/pull/19277#issuecomment-1111340350
and the uncaught exceptions.

I tested Node v16 with these changes and the test suite runs locally now.